modules/cnv-preparing-nodes-to-enable-dedicated-resources.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-preparing-nodes-to-enable-dedicated-resources_{context}"]
+
+= Preparing to enable dedicated resources
```

Comment:
Title needs update (if we're keeping it)

Reply:
We're scrapping this module.